src/NotificationServices/Softeq.NetKit.Services.SmsNotifications/Models/SmsDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+namespace Softeq.NetKit.Services.SmsNotifications.Models
+{
+    public class SmsDto
+    {
+        public string ToNumber { get; set; }
```

Comment:
Still not clear that name of the property. I would rename to RecipientPhoneNumber. And what if we need to send for multiple recipients?

Reply:
renamed and added multiple recipients